common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -455,3 +456,199 @@ def test_teams_membership(self):
+            'Create a new team if you can\'t find existing teams to join, '
+            'or if you would like to learn with friends you know.'
+        )
+        self.assertEqual(self.create_team_page.header_page_breadcrumbs, 'Example Topic')
+
+    def verify_and_navigate_to_create_team_page(self):
+        """Navigates to the create team page and verifies."""
+        self.create_team_page.wait_for_ajax()
+        self.assertTrue(self.browse_teams_page.create_team_link_present())
+        self.browse_teams_page.click_create_team_link()
+        self.verify_page_header()
+
+    def fill_create_form(self):
+        """Fill the create team form fields with appropriate values."""
+        self.create_team_page.value_for_text_field(field_id='name', value='Avengers')
+        self.create_team_page.value_for_textarea_field(
+            field_id='description',
+            value='The Avengers are a fictional team of superheroes.'
+        )
+        self.create_team_page.value_for_dropdown_field(field_id='language', value='English')
+        self.create_team_page.value_for_dropdown_field(field_id='country', value='Pakistan')
+
+    def test_user_can_see_create_team_page(self):
+        """
+        Scenario: The user should be able to see the create team page via teams list page.
+        Given I am enrolled in a course with a team configuration and a topic
+        When I visit the Teams page for that topic
+        Then I should see the Create Team page link on bottom
+        And When I click create team link
+        Then I should see the create team page.
+        And I should see the create team header
+        And I should also see the help messages for fields.
+        """
+        self.verify_and_navigate_to_create_team_page()
+        self.assertEqual(
+            self.create_team_page.message_for_field('name'),
+            'A name that identifies your team (maximum 255 characters).'
+        )
+        self.assertEqual(
+            self.create_team_page.message_for_field('country'),
+            'The country that team members primarily identify with.'
+        )
+        self.assertEqual(
+            self.create_team_page.message_for_field('language'),
+            'The language that team members primarily use to communicate with each other.'
+        )
+
+    def test_user_can_see_error_message_for_missing_data(self):
+        """
+        Scenario: The user should be able to see error message in case of missing required field.
+        Given I am enrolled in a course with a team configuration and a topic
+        When I visit the Create Team page for that topic
+        Then I should see the Create Team header and form
+        And When I click create team button without filling required fields
+        Then I should see the error message and highlighted fields.
+        """
+        self.verify_and_navigate_to_create_team_page()
+        self.create_team_page.create_team()
```

Comment:
This test case is too opaque. We navigate to the team page, create a team, and then expect to see an error? I suggest you rename `create_team` to `submit_form`, since `create_team` sounds like it should fill out the form as well. That way it is clear that we're seeing the error due to not filling out the form.